test_communication/test/message_fixtures.hpp

```diff
@@ -146,19 +146,19 @@ get_messages_dynamic_array_primitives()
   }
   {
     auto msg = std::make_shared<test_communication::DynamicArrayPrimitives>();
-    msg->bool_values = {{true}};
-    msg->byte_values = {{0xff}};
-    msg->char_values = {{'\255'}};
-    msg->float32_values = {{1.11f}};
-    msg->float64_values = {{1.11}};
-    msg->int8_values = {{127}};
-    msg->uint8_values = {{255}};
-    msg->int16_values = {{32767}};
-    msg->uint16_values = {{65535}};
-    msg->int32_values = {{2147483647}};
-    msg->uint32_values = {{4294967295}};
-    msg->int64_values = {{9223372036854775807}};
-    msg->uint64_values = {{18446744073709551615UL}};
+    msg->bool_values = {true};
```

Comment:
Will this change work on Windows? IIRC the whole point of using double braces was to make Visual Studio happy.